doc/rados/operations/erasure-code.rst

```diff
@@ -212,22 +212,23 @@ mode so that every write and read to the *ecpool* are actually using
 the *hot-storage* and benefit from its flexibility and speed.
 
 More information can be found in the `cache tiering
-<../cache-tiering>`_ documentation.
+<../cache-tiering>`_ documentation.  Note however that cache tiering
+is deprecated and may be removed completely in a future release.
 
 Erasure coded pool recovery
 ---------------------------
-If an erasure coded pool loses some shards, it must recover them from the others.
-This generally involves reading from the remaining shards, reconstructing the data, and
-writing it to the new peer.
-In Octopus, erasure coded pools can recover as long as there are at least *K* shards
+If an erasure coded pool loses some data shards, it must recover them from others.
+This involves reading from the remaining shards, reconstructing the data, and
+writing new shards.
+In Octopus and later releases, erasure coded pools can recover as long as there are at least *K* shards
 available. (With fewer than *K* shards, you have actually lost data!)
 
-Prior to Octopus, erasure coded pools required at least *min_size* shards to be
-available, even if *min_size* is greater than *K*. (We generally recommend min_size
-be *K+2* or more to prevent loss of writes and data.)
+Prior to Octopus, erasure coded pools required at least ``min_size`` shards to be
+available, even if ``min_size`` is greater than ``K``. We recommend ``min_size``
+be ``K+2`` or more to prevent loss of writes and data.
 This conservative decision was made out of an abundance of caution when designing the new pool
-mode but also meant pools with lost OSDs but no data loss were unable to recover and go active
-without manual intervention to change the *min_size*.
+mode but also meant that pools with lost OSDs but no data loss were unable to recover and go active
```

Comment:
s/but no data loss/and no data loss/

Making this conjunctional change will cause the "but" in "new pool mode but also" to unambiguously separate the sentence instead of putting a cognitive burden on the reader by making the reader disambiguate two "but"s.

Reply:
Ah.  I hadn't noticed the preceding instance.    I think the second `but` has value in that it emphasises OSDs being lost but not data, as opposed to situations where both are the case, i.e. OSDs failing in M+1 failure domains in parallel.  I've reworded to accommodate both our thoughts, PTAL.